### ISSUE TITLE
new skinchanger 

### ISSUE THREAD

**Issue by T9Jungler** [R3nzTheCodeGOD/R3nzSkin#1199]
https://discord.gg/Xu9ayvmCQs release tomorrow at 8AM (100% legit and free)

**Comment by lxccc812** [comment]
As free software you should open source him

**Comment by Lzx648740** [comment]
It is a free software ?

**Comment by breakertwisttwist** [comment]
> It is a free software ?

Yup and it works just fine

**Comment by Mzf12fs** [comment]
> > It is a free software ?
> 
> Yup and it works just fine

could u plz send me a new link so I can join the server?

**Comment by edudadz7** [comment]
Good evening, I'm from Brazil and I'm here to thank you for the app, it really works, I was unsure whether to add it or not, but it really works, thank you very much, and know that Brazil loves you

**Comment by Jarzom** [comment]
NOT SAFE:
https://www.virustotal.com/gui/file/f4f6c3526d139f3b31de55067de829fa788d64b4f10411ee2b5359f206b8edbf/detection

Hes even deleting Youtube comments about this. Do not install.

**Comment by breakertwisttwist** [comment]
> NOT SAFE: https://www.virustotal.com/gui/file/f4f6c3526d139f3b31de55067de829fa788d64b4f10411ee2b5359f206b8edbf/detection
> 
> Hes even deleting Youtube comments about this. Do not install.

youtube comments ? huh

**Comment by breakertwisttwist** [comment]
> NOT SAFE: https://www.virustotal.com/gui/file/f4f6c3526d139f3b31de55067de829fa788d64b4f10411ee2b5359f206b8edbf/detection
> 
> Hes even deleting Youtube comments about this. Do not install.

the virusTotal is not even my app LT SkinChanger😭😭

**Comment by lol123love** [comment]
> NOT SAFE: https://www.virustotal.com/gui/file/f4f6c3526d139f3b31de55067de829fa788d64b4f10411ee2b5359f206b8edbf/detection
> 
> Hes even deleting Youtube comments about this. Do not install.

huh ! ? that's another app that's not LT SkinChanger

**Comment by Mzf12fs** [comment]
so where i can find the SkinChanger you mentioned guys plz help

**Comment by 635882952** [comment]
![1718600728672](https://github.com/R3nzTheCodeGOD/R3nzSkin/assets/49578586/a7efe0d5-b256-4b73-a3f9-8d84a4593e79)

**Comment by illitiratehobo** [comment]
> ![1718600728672](https://private-user-images.githubusercontent.com/49578586/340184064-a7efe0d5-b256-4b73-a3f9-8d84a4593e79.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg2MTA5MzAsIm5iZiI6MTcxODYxMDYzMCwicGF0aCI6Ii80OTU3ODU4Ni8zNDAxODQwNjQtYTdlZmUwZDUtYjI1Ni00YjczLWEzZjktOGQ4NGE0NTkzZTc5LmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE3VDA3NTAzMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA5NDNjOWFjZjkxMDFkYjYyNjUxNjk5YmE5YzVhMTQwYWNiNTUyM2ZiMTYwYjg0MWM0YmU2ODNjZDk2YzJmZDgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.B0WkYoLo5qbJwT9kQEqVVtHVvdFtDI4TrhxJZ7LWeVM)

LMFAO r3nz has more brother 

![image](https://github.com/R3nzTheCodeGOD/R3nzSkin/assets/169540547/644ce9cf-a78c-41a6-b66c-910f9b841dde)

**Comment by 635882952** [comment]
> > ![1718600728672](https://private-user-images.githubusercontent.com/49578586/340184064-a7efe0d5-b256-4b73-a3f9-8d84a4593e79.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg2MTA5MzAsIm5iZiI6MTcxODYxMDYzMCwicGF0aCI6Ii80OTU3ODU4Ni8zNDAxODQwNjQtYTdlZmUwZDUtYjI1Ni00YjczLWEzZjktOGQ4NGE0NTkzZTc5LmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE3VDA3NTAzMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA5NDNjOWFjZjkxMDFkYjYyNjUxNjk5YmE5YzVhMTQwYWNiNTUyM2ZiMTYwYjg0MWM0YmU2ODNjZDk2YzJmZDgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.B0WkYoLo5qbJwT9kQEqVVtHVvdFtDI4TrhxJZ7LWeVM)
> 
> 哈哈哈哈 r3nz 有更多的兄弟
> 
> ![图像](https://private-user-images.githubusercontent.com/169540547/340223519-644ce9cf-a78c-41a6-b66c-910f9b841dde.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg2MTExMzAsIm5iZiI6MTcxODYxMDgzMCwicGF0aCI6Ii8xNjk1NDA1NDcvMzQwMjIzNTE5LTY0NGNlOWNmLWE3OGMtNDFhNi1iNjZjLTkxMGY5Yjg0MWRkZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYxN1QwNzUzNTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zZTA4NDY0ZjVhNjMzNjhiN2Y3MjRiYzY0MWIzOWE2NWE3YTI0NmI5YjExYzA4ZTQwYjZlNDBjOWU2MzIwYjZhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.ebdbSJV0hBJSEk345SGn0Q2pd8zrpPuylfWITwxmTbI)


But at least it's open source

**Comment by illitiratehobo** [comment]
> > > ![1718600728672](https://private-user-images.githubusercontent.com/49578586/340184064-a7efe0d5-b256-4b73-a3f9-8d84a4593e79.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg2MTA5MzAsIm5iZiI6MTcxODYxMDYzMCwicGF0aCI6Ii80OTU3ODU4Ni8zNDAxODQwNjQtYTdlZmUwZDUtYjI1Ni00YjczLWEzZjktOGQ4NGE0NTkzZTc5LmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE3VDA3NTAzMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA5NDNjOWFjZjkxMDFkYjYyNjUxNjk5YmE5YzVhMTQwYWNiNTUyM2ZiMTYwYjg0MWM0YmU2ODNjZDk2YzJmZDgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.B0WkYoLo5qbJwT9kQEqVVtHVvdFtDI4TrhxJZ7LWeVM)
> > 
> > 
> > 哈哈哈哈 r3nz 有更多的兄弟
> > ![图像](https://private-user-images.githubusercontent.com/169540547/340223519-644ce9cf-a78c-41a6-b66c-910f9b841dde.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg2MTExMzAsIm5iZiI6MTcxODYxMDgzMCwicGF0aCI6Ii8xNjk1NDA1NDcvMzQwMjIzNTE5LTY0NGNlOWNmLWE3OGMtNDFhNi1iNjZjLTkxMGY5Yjg0MWRkZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYxN1QwNzUzNTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zZTA4NDY0ZjVhNjMzNjhiN2Y3MjRiYzY0MWIzOWE2NWE3YTI0NmI5YjExYzA4ZTQwYjZlNDBjOWU2MzIwYjZhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.ebdbSJV0hBJSEk345SGn0Q2pd8zrpPuylfWITwxmTbI)
> 
> But at least it's open source

![image](https://github.com/R3nzTheCodeGOD/R3nzSkin/assets/169540547/679068b2-9148-424c-897f-7856d5c48487)


So I'm not an expert in cybersecurity or malware but I have some expertise in programming as a CS major. I'll do my best to simply explain why I'd advise against downloading this unless you can figure out how to build the program yourself (could maybe teach yourself with chatgpt).

I was able to compile and build the code directly from the source code rather than the precompiled source releases. Basically, the precompiled version is the one that they offer you with the "download" button.

The reason why this is important is that we have to trust that the precompiled version is from the same code that they openly listed on GitHub. R3nz (the owner) could hypothetically upload all the legit source code to gain trust while sneakily adding a virus to the precompiled version and you wouldn't be able to tell.

So I uploaded the precompiled exe and .dll as well as the ones I built to VirusTotal. From my *limited* understanding, the difference between the .dll and exe I built myself vs the one that was provided in the download page should basically be the same. That's why I find it very suspicious that the precompiled version is getting flagged more by virus total since they should basically be the same. Virus total score

Precompiled dll: https://www.virustotal.com/gui/file/5e5be7d8aa3a96bfb2a534897b2502799448886f402bc3378d5305a36349b1e1
Precompiled exe:
https://www.virustotal.com/gui/file/ea034d7b08a538f827293c3b0742d4c178708afdfd0f45d47cad99967b311a97
My compiled dll:
https://www.virustotal.com/gui/file/5fecf537cbe58d1d20e1fe4eb539537be6762bef8686721b523dd05d281850c4?nocache=1
My compiled exe:
https://www.virustotal.com/gui/file/4654a52d89ff1b6e364ff19dee3eead8aa3091e546929750cf18db465a6c651e?nocache=1

as you can see the two files which should basically be the exact same have wildly different scores on VirusTotal. I just find it kind of weird and while there could be a perfectly good explanation I can't really give one as I'm not knowledgeable enough in this field.

TLDR: I would strongly err on the side of caution if you're unable to compile the exe and dll yourself.

**Comment by lol123love** [comment]
> > > > ![1718600728672](https://private-user-images.githubusercontent.com/49578586/340184064-a7efe0d5-b256-4b73-a3f9-8d84a4593e79.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg2MTA5MzAsIm5iZiI6MTcxODYxMDYzMCwicGF0aCI6Ii80OTU3ODU4Ni8zNDAxODQwNjQtYTdlZmUwZDUtYjI1Ni00YjczLWEzZjktOGQ4NGE0NTkzZTc5LmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE3VDA3NTAzMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA5NDNjOWFjZjkxMDFkYjYyNjUxNjk5YmE5YzVhMTQwYWNiNTUyM2ZiMTYwYjg0MWM0YmU2ODNjZDk2YzJmZDgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.B0WkYoLo5qbJwT9kQEqVVtHVvdFtDI4TrhxJZ7LWeVM)
> > > 
> > > 
> > > 哈哈哈哈 r3nz 有更多的兄弟
> > > ![图像](https://private-user-images.githubusercontent.com/169540547/340223519-644ce9cf-a78c-41a6-b66c-910f9b841dde.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg2MTExMzAsIm5iZiI6MTcxODYxMDgzMCwicGF0aCI6Ii8xNjk1NDA1NDcvMzQwMjIzNTE5LTY0NGNlOWNmLWE3OGMtNDFhNi1iNjZjLTkxMGY5Yjg0MWRkZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYxN1QwNzUzNTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zZTA4NDY0ZjVhNjMzNjhiN2Y3MjRiYzY0MWIzOWE2NWE3YTI0NmI5YjExYzA4ZTQwYjZlNDBjOWU2MzIwYjZhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.ebdbSJV0hBJSEk345SGn0Q2pd8zrpPuylfWITwxmTbI)
> > 
> > 
> > But at least it's open source
> 
> ![image](https://private-user-images.githubusercontent.com/169540547/340370107-679068b2-9148-424c-897f-7856d5c48487.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg2NDc3MzAsIm5iZiI6MTcxODY0NzQzMCwicGF0aCI6Ii8xNjk1NDA1NDcvMzQwMzcwMTA3LTY3OTA2OGIyLTkxNDgtNDI0Yy04OTdmLTc4NTZkNWM0ODQ4Ny5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYxN1QxODAzNTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03NGY1MTk2NTkyNDI5YjE4NzE1ZmI4ODQ1NWU2ZmM0MTRiMTdhYWZiOTA0YTg4YWM2NjhlMDgzNGJjOWVkZjA5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.6oWH1rrtXbfBotOMSuMOpWRkIb_03FZEMW3_0SMh9-k)
> 
> So I'm not an expert in cybersecurity or malware but I have some expertise in programming as a CS major. I'll do my best to simply explain why I'd advise against downloading this unless you can figure out how to build the program yourself (could maybe teach yourself with chatgpt).
> 
> I was able to compile and build the code directly from the source code rather than the precompiled source releases. Basically, the precompiled version is the one that they offer you with the "download" button.
> 
> The reason why this is important is that we have to trust that the precompiled version is from the same code that they openly listed on GitHub. R3nz (the owner) could hypothetically upload all the legit source code to gain trust while sneakily adding a virus to the precompiled version and you wouldn't be able to tell.
> 
> So I uploaded the precompiled exe and .dll as well as the ones I built to VirusTotal. From my _limited_ understanding, the difference between the .dll and exe I built myself vs the one that was provided in the download page should basically be the same. That's why I find it very suspicious that the precompiled version is getting flagged more by virus total since they should basically be the same. Virus total score
> 
> Precompiled dll: https://www.virustotal.com/gui/file/5e5be7d8aa3a96bfb2a534897b2502799448886f402bc3378d5305a36349b1e1 Precompiled exe: https://www.virustotal.com/gui/file/ea034d7b08a538f827293c3b0742d4c178708afdfd0f45d47cad99967b311a97 My compiled dll: https://www.virustotal.com/gui/file/5fecf537cbe58d1d20e1fe4eb539537be6762bef8686721b523dd05d281850c4?nocache=1 My compiled exe: https://www.virustotal.com/gui/file/4654a52d89ff1b6e364ff19dee3eead8aa3091e546929750cf18db465a6c651e?nocache=1
> 
> as you can see the two files which should basically be the exact same have wildly different scores on VirusTotal. I just find it kind of weird and while there could be a perfectly good explanation I can't really give one as I'm not knowledgeable enough in this field.
> 
> TLDR: I would strongly err on the side of caution if you're unable to compile the exe and dll yourself.

Damnnn that's actually crazyy

**Comment by rithik370** [comment]
> https://discord.gg/Xu9ayvmCQs release tomorrow at 8AM (100% legit and free)

bro the link is expired can u send it again?
